src/plugin/plugin_processing_behaviorTriggeredAverage.py

```diff
@@ -0,0 +1,479 @@
+from qtpy.QtWidgets import QFrame, QMessageBox, QCheckBox, QMenu, QFileDialog, QLabel
+from plugin.behaviorTriggeredAverage_ui import Ui_BTAFrame
+from qtpy.QtCore import Slot
+from os.path import expanduser, sep
+from qtpy.QtGui import QColor
+import numpy as np
+import pandas as pd
+from scipy import signal
+import math
+from processing.processing import ProcessingBase
+from vispy.scene import SceneCanvas, visuals, AxisWidget
+from vispy.visuals.transforms import STTransform
+from PIL import Image
+
+
+class behaviorTriggeredAverage(QFrame, ProcessingBase):
+    def __init__(self, nwbFile, bento):
+        QFrame.__init__(self)
+        ProcessingBase.__init__(self, nwbFile, bento)
+        self.bento = bento
+        self.checkData()
+        if not self.annotationsExists or not self.neuralExists:
+            msgBox = QMessageBox(QMessageBox.Warning, 
+                                 "Required Data not found", 
+                                 "Either Neural data or Annotation data \
```

Comment:
change to "both neural data and annotation data must exist for this plugin to work"